app/components/pipeline/jobs/table/component.js

```diff
@@ -26,8 +26,14 @@ export default class PipelineJobsTableComponent extends Component {
     },
     {
       title: 'HISTORY',
+      propertyName: 'history',
       className: 'history-column',
-      component: 'historyCell'
+      component: 'historyCell',
+      filterFunction: async function (_, filterVal) {
+        this.dataReloader.setNumBuilds(filterVal);
+      }.bind(this),
```

Comment:
```suggestion
      filterFunction: async (_, filterVal) => {
        await this.dataReloader.setNumBuilds(filterVal);
      },
```
If you use the arrow function you don't have to manually bind